eZ/Publish/Core/settings/search_engines/legacy/criterion_handlers_common.yml

```diff
@@ -54,6 +54,12 @@ services:
             - "@ezpublish.api.storage_engine.transformation_processor"
 
     # Criterion handlers
+    ezpublish.search.legacy.gateway.criterion_handler.common.aggregate:
```

Comment:
Based on my experience with handling that particular debt I can already tell you that I'm not gonna allow non-FQCN-named services, regardless of the context it's put in. Unless there's a technical reason (like multiple implementations of the same class).

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2958/commits/f5b574533d90c7f342beefeca599f60bf1ecbdd0